### PR TITLE
allow Users&Groups and Org Details with no projects.

### PR DESCRIPTION
**Pull request by sarahentzel**
set default project to saved last project if available

### CODE REVIEW

File: src/components/Invite.tsx [hunk 1]
```diff
@@ -305,17 +305,19 @@ function Invite(props: IProps) {
...
-        ? list.substring(0, list.length - 2)
-        : t.noProjects
-    );
+    if (allusersgroup.length > 0) {
+      setAllUsersGroup(allusersgroup[0].id);
+      var assocProjects = projects
+        .filter(p => related(p, 'group') === allusersgroup[0].id)
+        .map(p => p.attributes.name);
+      var list = '';
+      assocProjects.forEach(p => (list += p + ', '));
```
Comment (gtryus) [inline src/components/Invite.tsx]: I think these two lines can be simplifies:
const list = assocProjects.join(', ');

File: src/components/Invite.tsx [hunk 2]
```diff
@@ -305,17 +305,19 @@ function Invite(props: IProps) {
...
+      setAllUsersProjects(
+        assocProjects.length > 0
+          ? list.substring(0, list.length - 2)
```
Comment (gtryus) [inline src/components/Invite.tsx]: using join as above means this line can just use list.

File: src/routes/Loading.tsx [hunk 3]
```diff
@@ -267,32 +276,34 @@ export function Loading(props: IProps) {
...
-        !uiLang.includes(userRec.attributes.locale)
-      ) {
-        return <Redirect to="/profile" />;
+      if (userRec.attributes) {
+        if (!hasAnyRelated(userRec, 'groupMemberships')) {
+          const orgRec: Organization = {
+            type: 'organization',
+            attributes: {
+              name: t.myWorkbench,
+              description: 'Default organization of ' + userRec.attributes.name,
```
Comment (gtryus) [inline src/routes/Loading.tsx]: We should localize this string

File: src/components/Invite.tsx (hunk 1 above)
Comment (sarahentzel) [reply src/components/Invite.tsx]: awesome!

**Pull request merged**